src/responder/nss/nsssrv.c

```diff
@@ -540,6 +541,20 @@ int main(int argc, const char *argv[])
     /* set up things like debug, signals, daemonization, etc... */
     debug_log_file = "sssd_nss";
 
+    if (is_socket_activated()) {
+        char *logpath;
+        int fd;
+
+        ret = asprintf(&logpath, "%s/%s.log", LOG_PATH, debug_log_file);
+        if (ret == -1) {
+            return ENOMEM;
+        }
+
+        fd = open(logpath, O_RDONLY);
+        fchown(fd, 0, 0);
+        close(fd);
+    }
+
```

Comment:
Hmm. Seems that I can actually set up the log rotation later on (as done nowadays).